src/components/30-textarea/textarea.config.yml

```diff
@@ -0,0 +1,30 @@
+title: "Textarea"
+status: "ready"
+collated: true
+context:
+    alert: false
+variants:
+    - name: "disabled"
```

Comment:
Once you've made the updates to the templates, we'll need to add `id` fields to all of the `context` objects.